pkg/webui/console/components/live-data-button/index.js

```diff
@@ -0,0 +1,51 @@
+// Copyright © 2024 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React, { useCallback } from 'react'
+import { defineMessages } from 'react-intl'
+import PropTypes from 'prop-types'
+
+import Icon, { IconChevronUp } from '@ttn-lw/components/icon'
+
+import Message from '@ttn-lw/lib/components/message'
+
+import style from './live-data-button.styl'
+
+const m = defineMessages({
+  liveData: 'Live Data',
+})
+
+const LiveDataButton = props => {
+  const { setIsOpen } = props
+
+  const handleClick = useCallback(() => {
+    setIsOpen(true)
+  }, [setIsOpen])
+
+  return (
+    <div className={style.container}>
+      <button className={style.button} onClick={handleClick}>
+        <Icon className={style.icon} icon={IconChevronUp} />
+        <Message content={m.liveData} className={style.message} component={'span'} />
```

Comment:
```suggestion
        <Message content={m.liveData} className={style.message} component="span" />
```
I know we do `{'something'}` in several places, but let's avoid that.